docs/getting-started/quick-start.ipynb

```diff
@@ -20,7 +20,8 @@
     "import scipp as sc\n",
     "from scipp import Dim\n",
     "from scipp.plot import plot\n",
-    "sc.plot.config.backend = 'static'"
+    "sc.plot.config.backend = 'matplotlib:quiet'\n",
```

Comment:
There are some changes here and below that don't seem relevant to this PR. Accidentally staged?

Reply:
Thanks. Gone now that I fixed the base branch.